gramps/plugins/tool/check.py

```diff
@@ -1065,16 +1056,15 @@ def _fx(value):
             if len(self.empty_objects[the_type]) == 0:
                 logging.info("    OK: no empty %s found", the_type)
 
-    def _check_empty(self, data, empty_data, changepos):
+    def _check_empty(self, data, empty_data):
         """compare the data with the data of an empty object
         change, handle and gramps_id are not compared"""
-        if changepos is not None:
-            return (
-                data[2:changepos] == empty_data[2:changepos]
-                and data[changepos + 1 :] == empty_data[changepos + 1 :]
-            )
-        else:
-            return data[2:] == empty_data[2:]
+        for key in empty_data:
+            if key in ["change", "gramps_id", "handle"]:
```

Comment:
Is it worth extracting ["change", "gramps_id", "handle"] and giving it a name to describe why these keys are skipped e.g. internal_data_keys = ["change", "gramps_id", "handle"]

Reply:
I think I'll leave it like this as the only note about it is in the comments here, and I think this makes it clear. Also, as far as I saw, it was used only here.